### PR TITLE
FTUE - Account created screen

### PR DESCRIPTION
**Pull request by ouchadam**
Part of #4586 https://github.com/vector-im/element-meta/issues/132

- Adds an account created screen behind a feature flag
- The `personalise profile` button TODO is handled by https://github.com/vector-im/element-android/pull/5211

| SCREEN | FEATURE FLAG | SYSTEM BACK | TAKE ME HOME | 
| --- | --- | --- | --- |
|Screenshot_20220204_171024|Screenshot_20220204_172701|after-system-back|after-take-me-home

### CODE REVIEW

**Review by ganfra — APPROVED** [review]
Thanks, LGTM, just one remark!

File: vector/src/main/java/im/vector/app/features/onboarding/OnboardingViewModel.kt [hunk 1]
```diff
@@ -147,6 +147,8 @@ class OnboardingViewModel @AssistedInject constructor(
             is OnboardingAction.UserAcceptCertificate      -> handleUserAcceptCertificate(action)
             OnboardingAction.ClearHomeServerHistory        -> handleClearHomeServerHistory()
             is OnboardingAction.PostViewEvent              -> _viewEvents.post(action.viewEvent)
+            OnboardingAction.PersonalizeProfile            -> _viewEvents.post(OnboardingViewEvents.OnPersonalizeProfile)
```
[the remark this answers is not in the document]
Comment (ouchadam) [reply vector/src/main/java/im/vector/app/features/onboarding/OnboardingViewModel.kt:150]: yeah I wasn't sure about this as well, sounds like I should use `OnboardingAction.PostViewEvent` which acts as a proxy to post events directly to the `ViewModel.viewEvents` observer

[the remark this answers is not in the document]
Comment (ganfra) [reply vector/src/main/java/im/vector/app/features/onboarding/OnboardingViewModel.kt]: Thanks!

File: vector/src/main/res/values/donottranslate.xml [hunk 2]
```diff
@@ -16,4 +16,10 @@
 
     <!-- onboarding english only word play -->
     <string name="cut_the_slack_from_teams" translatable="false">Cut the slack from teams.</string>
+
+    <!--  WIP -->
+    <string name="ftue_account_created_personalize" translatable="false">Personalise profile</string>
```
Comment (bmarty) [inline vector/src/main/res/values/donottranslate.xml:21]: Default is EN-US so it has to be
```suggestion
    <string name="ftue_account_created_personalize" translatable="false">Personalize profile</string>
```

**Review by bmarty — APPROVED** [review]
Small small remarks. Please merge once handled. Thanks!

File: vector/src/main/java/im/vector/app/features/onboarding/ftueauth/FtueAuthAccountCreatedFragment.kt [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        views.accountCreatedTakeMeHome.setOnClickListener { viewModel.handle(OnboardingAction.PostViewEvent(OnboardingViewEvents.OnTakeMeHome)) }
+    }
+
+    override fun resetViewModel() {
+        // Nothing to do
+    }
+
+    override fun onBackPressed(toolbarButton: Boolean): Boolean {
+        viewModel.handle(OnboardingAction.PostViewEvent(OnboardingViewEvents.OnTakeMeHome))
+        return false
```
[the remark this answers is not in the document]
Comment (ouchadam) [reply vector/src/main/java/im/vector/app/features/onboarding/ftueauth/FtueAuthAccountCreatedFragment.kt]: ah great catch! 4975406e66efd0ef9ee75891e8aae46f503e3dd2

File: vector/src/main/java/im/vector/app/features/onboarding/ftueauth/FtueAuthAccountCreatedFragment.kt [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        setupViews()
+    }
+
+    private fun setupViews() {
+        views.accountCreatedSubtitle.text = getString(R.string.ftue_account_created_subtitle, activeSessionHolder.getActiveSession().myUserId)
+        views.accountCreatedPersonalize.setOnClickListener { viewModel.handle(OnboardingAction.PostViewEvent(OnboardingViewEvents.OnPersonalizeProfile)) }
```
[the remark this answers is not in the document]
Comment (ouchadam) [reply vector/src/main/java/im/vector/app/features/onboarding/ftueauth/FtueAuthAccountCreatedFragment.kt]: :+1:  updated 182dc2a098f4a3c00b85a111fcb4d5bd51059114

File: vector/src/main/res/values/donottranslate.xml (hunk 2 above)
Comment (ouchadam) [reply vector/src/main/res/values/donottranslate.xml]: :+1:  will also let design/product know 06f2ca1379c0fb35fc6a3f5b0309cd6e7e4f3470

**Review by bmarty — APPROVED** [review]
Thanks for the update!

**Pull request merged**